test/parser/parser_test.cpp

```diff
@@ -417,9 +417,13 @@ TEST_F(ParserTests, WrongQueryTest) {
   std::vector<std::string> queries;
   queries.push_back("SELECT;");
```

Comment:
This might actually be valid SQL. It works in Postgres, but not MySQL or SQLite.

Reply:
Yeah, but if I remember correctly there was an issue about this query before, at that time we added this WrongQueryTest.